src/containers/FourFieldForm.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+// Components
+import SubmitButton from '../components/submitButton';
+import InputField from '../components/input-field';
+
+// helpers
+// TODO: not sure how to fix this.
+import submitForm from '../helpers/index';
+
+const FourFieldForm = () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [company, setCompany] = useState('');
+  const [phone, setPhone] = useState('');
+  // Unable to pass this inline as attribute value.
+  const values = {
+    name, email, company, phone,
+  };
```

Comment:
Unable to pass these values inline,

Reply:
Not sure what you mean here

Reply:
So, if we try to pass the values directly into the submit component
```
 <SubmitButton action = {submitForm} values= {values} />
<SubmitButton action = {submitForm} values= { name, email, company, phone,} />
```
I get the following eslint error https://eslint.org/docs/rules/no-sequences